packages/desmos-std/src/profiles/models_common.rs

```diff
@@ -0,0 +1,10 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub struct PubKey {
+    #[serde(rename = "@type")]
+    pub proto_type: String,
+    pub key: String,
```

Comment:
```suggestion
    pub key: Binary,
```

The key from desmos chain is base64 encoded. It could be Binary type.